app/helpers/application_helper.rb

```diff
@@ -32,6 +33,32 @@ def link_to(*args, &block)
     end
   end
 
+  def filter_turbolinks(args)
```

Comment:
Watch out, `link_to` arguments are shifted when block is passed. See the implementation in http://apidock.com/rails/ActionView/Helpers/UrlHelper/link_to I'd say, for simplicity's sake, it's ok to register just regex or string and match against `url_for` for the link that's being created.